.github/workflows/tests.yml

```diff
@@ -232,8 +232,10 @@ jobs:
           bottles-directory: ${{ env.BOTTLES_DIR }}
           cleanup: ${{ matrix.cleanup }}
 
-      - run: brew test-bot ${{ needs.setup_tests.outputs.test-bot-formulae-args }}
-        id: brew-test-bot-formulae
+      - id: brew-test-bot-formulae
+        run: brew test-bot $TEST_BOT_FORMULAE_ARGS
```

Comment:
`shellcheck` wants `$TEST_BOT_FORMULAE_ARGS` to be quoted but I'm not sure if this would work the same way if it's quoted. Same with `$TEST_BOT_DEPENDENTS_ARGS` below.

If someone can confirm, I'll either quote these or add `# shellcheck disable=SC2086` in these areas.

Reply:
It should work since we're using Bash here (on the contrary, the same doesn't seem to work for Zsh):

```console
$ args="info hello"
$ brew $args
==> hello: stable 2.12.1 (bottled)
Program providing model for GNU coding standards and practices
https://www.gnu.org/software/hello/
Conflicts with:
  perkeep (because both install `hello` binaries)
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/h/hello.rb
License: GPL-3.0-or-later
==> Analytics
install: 11,441 (30 days), 15,743 (90 days), 35,030 (365 days)
install-on-request: 11,441 (30 days), 15,743 (90 days), 35,028 (365 days)
build-error: 11 (30 days)
```

Quoting wouldn't work as the variable will be treated as a single argument. So I think we should be adding a `shellcheck disable` here.

Reply:
That's what I was seeing when working on similar changes in other workflows, so thanks for confirming. I'll update this to add `shellcheck` comments for now.